ansible/roles/jenkins/tasks/gcloud.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: Install gcloud apt deps
+  ansible.builtin.apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - apt-transport-https
+    - ca-certificates
+    - gnupg
+  become: true
+
+- name: Add gcloud apt key
+  ansible.builtin.apt_key:
+    url: "{{ gcp.gcloud_apt_key }}"
+    state: present
+  become: true
+
+- name: Add gcloud apt repo
+  ansible.builtin.apt_repository:
+    repo: "{{ gcloud_apt_repo }}"
+    state: present
+  become: true
+
+- name: Install gcloud
+  ansible.builtin.apt:
+    name: google-cloud-cli
+    state: present
+  become: true
+
+- name: Configure gcloud for jenkins account
+  become: true
+  become_user: jenkins
+  block:
+    - name: Activate service account
+      ansible.builtin.command: gcloud auth activate-service-account --key-file={{ gcp.service_account_key_file }}
```

Comment:
Service account url argument and --project option are missing

Reply:
For reference :
https://github.com/TEAM-AAAAAAAAAAAAAAAA/Whanos/blob/main/scripts/gcloud-setup.sh#L10

Reply:
My bad